language/En.txt

```diff
@@ -121,7 +121,7 @@ CUSTOM_POG_LIBRARY          = Pog Library
 //Map -> Grid Mode
 MAP_CLEAR                   = Clear map
 MAP_CLEAR_WARNING           = This will clear all lines, pogs, and underlays on the entire layer. Are you sure?
-MAP_GRID_CHANGE             = changes the grid mode.
+MAP_GRID_CHANGE             = %s changes the grid mode.
```

Comment:
I pondered that for a while. I decided to do it in the end because this is a localisation file, and the constant is used for localising a message. When you do localisation, you localise an entire phrase, not just part of it - different languages may put the player name elsewhere in the phrase.
I'm afraid I don't know the best way of doing it in Java. You could have an interface for the phrases and each locale implements it and does what they wish. I suspect someone has already had this problem though and come up with a standard solution.